cluster/aws/templates/common.sh

```diff
@@ -15,11 +15,15 @@
 # limitations under the License.
 
 
-apt-get update
-apt-get install --yes curl
+until apt-get update; do
+  echo "== apt-get update failed, retrying =="
+  echo sleep 5
+done
+
+apt-get install -y curl
+apt-get dist-upgrade -y
 
 # Retry a download until we get it.
-#
```

Comment:
If you're worried about script size, these lines are actually stripped before upload.  I don't think we're particularly strong on bash-style here, so either way is fine.